rpcs3/Emu/System.cpp

```diff
@@ -72,6 +74,8 @@ atomic_t<s64> g_tty_size{0};
 std::array<std::deque<std::string>, 16> g_tty_input;
 std::mutex g_tty_mutex;
 
+named_thread<IPC_server> m_ipc_server;
```

Comment:
This should probably either be fxo or a member of Emulator.
Currently, the "m_" prefix is wrong in this location and should be renamed to "g_".

Reply:
I've just gone ahead and changed the prefix for now, moving it is a headache for another day.